content/src/main/content/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/clientlibs/notification.js

```diff
@@ -50,9 +50,9 @@ $(function() {
     });
     
     function setSessionCookie(name, value) {
```

Comment:
minor nit - this function is incorrectly named. We aren't setting a "session" cookie. Quite the opposite; we are setting a far future cookie.